src/Core/src/Fonts/EmbeddedFontLoader.Windows.cs

```diff
@@ -57,5 +89,68 @@ static string CleanseFilePath(string filePath)
 
 			return relativePath;
 		}
+
+		static string CleanseUnpackagedFilePath(string filePath)
+		{
+			// yes, ms-appx, yes, it is the way... sometimes #acceptance
+			//
+			// The ms-appx prefix is used for fonts, regardless of the location,
+			// whether in the package or in the local file system.
+			// The times it is not used is when it is in another, special
+			// location, such as in the app data.
+
+			var baseUri = new Uri("ms-appx://");
+			var uri = new Uri(baseUri, "ms-appx://" + filePath);
+			var relativePath = uri.ToString().TrimEnd('/');
+
+			return relativePath;
+		}
+
+		private string? LoadFontUnpackaged(string fontName, Stream resourceStream)
+		{
+			var tmpdir = Path.Combine(FileSystem.AppDataDirectory, "..", FontCacheFolderName);
```

Comment:
Should this be `FileSystem.Current.CacheDirectory` ?

Reply:
I will change it for clarity, but we are actually going up a level, so it is not really cache nor app data. It is basically a cache, but a private one.

Reply:
Hmm, I'll have to fix that in a separate PR and deal with migrations and things :(